lua/yanky.lua

```diff
@@ -237,8 +237,10 @@ function yanky.cycle(direction)
 end
 
 function yanky.on_yank()
-  if "_" == vim.v.register then
-    return
+  for _k, v in pairs(yanky.config.options.ring.ignore_registers) do
```

Comment:
`_k` is unused, please use `_` as variable name instead

```suggestion
  for _, v in pairs(yanky.config.options.ring.ignore_registers) do
```